CONTRIBUTING.md

```diff
@@ -181,38 +181,31 @@ JavaScript and TypeScript files are validated using a combination of [Prettier](
 To release a stable version, go on `master` (`git checkout master`) and use:
 
 ```sh
-npm run release
+npm run release:prepare
 ```
 
-_Make sure to use `npm run` instead of `yarn run` to avoid issues._
+It will create a pull-request for next release. When it's reviewed, approved and merged, then CircleCI will automatically publish it to NPM.
 
-### Maintenance version
+### Maintenance version (v3 or below)
 
-For the maintenance version, go on maintenance (`git checkout maintenance`) and use:
+For the maintenance version, go on a maintenance branch(e.g., `git checkout maintenance-v3`) and use:
```

Comment:
Wdyt about removing the `maintenance-` prefix? I don't see any benefit of having it.

```suggestion
For the maintenance versions, go to a previous version branch (e.g., `git checkout v3`) and use:
```